packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -4,6 +4,68 @@ title: Okta Identity Engine API Products release notes 2022
 
 <ApiLifecycle access="ie" />
 
+## August
+
+### Monthly release 2022.08.0
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [PKCE validation for OIDC app integrations is GA in Preview](#pkce-validation-for-oidc-app-integrations-is-ga-in-preview) | July 7, 2022|
```

Comment:
The order of the entries seems random. Didn't we consider ordering them with GA first? Or by the date they hit the preview orgs?

Reply:
@brentschaus-okta I recall we had a discussion about ordering but never came to any decision. I believe I dealt with them as they appeared in the wiki, although as tickets come in and go out, it even makes that "ordering" difficult. As this is a fairly large release, I'd rather not be cutting and pasting at this stage. Okay to move this to a team discussion?

Reply:
Added a comment on the Files Changed page. For now, I'd like to avoid re-ordering these notes as we're close to release.